quarkus/extensions/src/main/java/org/keycloak/provider/quarkus/QuarkusCacheManagerProvider.java

```diff
@@ -44,12 +44,13 @@
         try {
             InputStream configurationStream = loadConfiguration(config);
             ConfigurationBuilderHolder builder = new ParserRegistry().parse(configurationStream);
+            boolean isClustered = builder.getNamedConfigurationBuilders().get("sessions").clustering().cacheMode().isClustered();
 
-            if (builder.getNamedConfigurationBuilders().get("sessions").clustering().cacheMode().isClustered()) {
+            if (isClustered) {
                 configureTransportStack(config, builder);
             }
 
-            return (C) new DefaultCacheManager(builder, true);
+            return (C) new DefaultCacheManager(builder, isClustered);
         } catch (Exception e) {
```

Comment:
Does this have anything to do with JPA or Liquibase?

Reply:
No. But I also changed that to avoid starting the cache beforehand if running a local cache. Sorry for including it.